test/e2e/self_provisioned_elasticsearch_test.go

```diff
@@ -71,6 +67,7 @@ func (suite *SelfProvisionedTestSuite) SetupTest() {
 }
 
 func (suite *SelfProvisionedTestSuite) TestSelfProvisionedESSmokeTest() {
+	t.Skip("Skipping until issue481 is fixed")
```

Comment:
@kevinearls add a note to that issue that this should be enabled back. Otherwise it will be forgotten!